### PR TITLE
Trev8939/scene layer selection

### PR DESCRIPTION
**Pull request by TADraeseke**
add sample as per: https://devtopia.esri.com/runtime/common-samples/issues/655

### CODE REVIEW

**Comment by TADraeseke** [comment]
@eri9600 - Can you review please? Thanks!

File: java/scene-layer-selection/src/main/java/com/esri/arcgisruntime/scenelayerselection/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
+package com.esri.arcgisruntime.scenelayerselection;
```
Comment (eri9600) [inline java/scene-layer-selection/src/main/java/com/esri/arcgisruntime/scenelayerselection/MainActivity.java]: The copyright is missing at the top

File: java/scene-layer-selection/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Scene Layer Selection
+Select geoelements in a scene layer.
+
+![Scene Layer Select GeoElement App](scene-layer-selection.png)
```
Comment (eri9600) [inline java/scene-layer-selection/README.md]: The image description should be similar to the module/image name. I think it should be **Scene Layer Selection App**

File: java/scene-layer-selection/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <string name="app_name">Scene Layer Selection</string>
+    <string name="brest_buildings">
+        "http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer"</string>
```
Comment (eri9600) [inline java/scene-layer-selection/src/main/res/values/strings.xml]: I don't think the quotations are necessary.

**Comment by TADraeseke** [comment]
Thanks @eri9600 for that rapid review :)

File: java/scene-layer-selection/src/main/java/com/esri/arcgisruntime/scenelayerselection/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    mSceneView = findViewById(R.id.sceneView);
+    mSceneView.setScene(scene);
+
+    // add base surface with elevation data
+    Surface surface = new Surface();
+    final String elevationService = getString(R.string.world_elevation_url);
+    surface.getElevationSources().add(new ArcGISTiledElevationSource(elevationService));
+    scene.setBaseSurface(surface);
+
+    // add a scene layer of Harvard buildings to the scene
```
Comment (eri9600) [inline java/scene-layer-selection/src/main/java/com/esri/arcgisruntime/scenelayerselection/MainActivity.java]: Hey @TADraeseke, Tyler pointed out that we need to update the comment to reflect the Brest layer not Harvard.

**Pull request merged**